transformer_engine/pytorch/module/layernorm_linear.py

```diff
@@ -428,7 +428,7 @@ def forward(
                 fsdp_group,
                 mu,
                 rsigma,
-                weightmat if is_weight_param_quantized else None,
+                weightmat if not is_weight_param_quantized else None,
```

Comment:
Change to match previous behavior and behavior in `Linear`:
```suggestion
                weightmat if fp8 and not is_weight_param_quantized else None,
```